README.md

```diff
@@ -136,3 +143,5 @@ Contributions are welcome.
 
 This project has come out of the work at [hasura.io](https://hasura.io). 
 Current maintainers [@Tirumarai](https://twitter.com/Tirumarai), [@shahidh_k](https://twitter.com/shahidh_k). 
+
```

Comment:
there is a footer section in the layouts folder which has the twitter link . so we don't need to add this?

Also, can you change the discord link in the footer as well?